opentelemetry-sdk/Cargo.toml

```diff
@@ -51,6 +51,9 @@ testing = ["opentelemetry/testing", "trace", "metrics", "logs", "rt-async-std",
 rt-tokio = ["tokio", "tokio-stream"]
 rt-tokio-current-thread = ["tokio", "tokio-stream"]
 rt-async-std = ["async-std"]
+memory-profiling = []
```

Comment:
Do we want this to be a crate feature?

Reply:
Good catch, leftover of mem stats I though of adding earlier. Removed now.